suite-native/module-connect-device/src/components/ConnectDeviceScreenHeader.tsx

```diff
@@ -33,14 +45,18 @@ export const ConnectDeviceScreenHeader = () => {
 
     return (
         <ScreenHeaderWrapper>
-            <IconButton
-                iconName="close"
-                size="medium"
-                colorScheme="tertiaryElevation1"
-                accessibilityRole="button"
-                accessibilityLabel="close"
-                onPress={handleCancel}
-            />
+            <Box>
+                {shouldDisplayCancelButton && (
```

Comment:
nit: the `Box` was not necessary before. Isn't it redundant?

Reply:
It's needed for when parents flexbox calculates the spacings. If no icon is there and somebody puts icon to the right, flexbox alignments will be off, so I just added it.